### ISSUE TITLE
Out of date controlled vocabularies for IPT instalation

### ISSUE THREAD

**Issue by RicardoOrtizG** [gbif/ipt#1671]
Hi!
I'm trying to publish Material Sample datasets with basisOfRecord documented as "MaterialSample", but the IPT show me an error due to controlled vocabularie, suggesting me to translate to "Sample". According to TDWG the vocabularie corresponds to "MaterialSample", I try to actualize the extensions in the IPT using the Synchronise Extensions and Vocabularies button, but the issue remains. 

The IPTs extension documentation shows the option Sample under versión 2021-07-15
![imagen](https://user-images.githubusercontent.com/15789737/138907848-990432cb-eba6-435e-9acb-8a1edb69cbd3.png)

IPT version: IPT Versión 2.4.0-r6f4f375
URL: ipt.biodiversidad.co/iavh/

I look for the same problem in other IPT's and the vocabularies differ as show below andunder the same versión 2021-07-15
![imagen](https://user-images.githubusercontent.com/15789737/138907506-b5844fa8-850b-4005-ac82-dc5b3b718bc1.png)

¿Can you tell me what can I do to synchronize the vocabularies? thanks in advance.

**Comment by RicardoOrtizG** [comment]
Hi Mike, 

I see the issue you attach to mine and decide to wait for the update of the IPT to the 2.5.1 version, unfortunelly it is not working also. Sitll have the same problem.

![imagen](https://user-images.githubusercontent.com/15789737/142495506-87ab572b-3115-4244-bdea-d286b95b2347.png)

So, the only possible solution is to removing the vocabulary file from the IPT? or maybe is a bug that maybe will be solved in 2.5.2 version?

@camiplata

**Comment by mike-podolskiy90** [comment]
Thank you for your message @RicardoOrtizG 
Could I have a look at the following files: `/datadir/config/.extensions/http_rs_tdwg_org_dwc_terms_Occurrence.xml` and `http_rs_gbif_org_vocabulary_dwc_basis_of_record_xml.vocab` from your IPT?

**Comment by MattBlissett** [comment]
In the first screenshot the URI is http://rs.tdwg.org/dwc/dwctype/Sample.

I don't know where that could have come from.  It's not in https://rs.gbif.org/vocabulary/dwc/basis_of_record.xml , or anywhere else in rs.gbif.org.  Is the issue (emitido) date 13 February 2015?

**Comment by RicardoOrtizG** [comment]
Hi, Im sharing with you the files you asked. 
[IPTVocabularyFiles.zip](https://github.com/gbif/ipt/files/7590883/IPTVocabularyFiles.zip)

**Comment by MattBlissett** [comment]
Are those files from the IPT showing "Sample"?

This is even more strange, since the file contains the correct definition:

``` <concept dc:identifier="MaterialSample" 
	 dc:URI="http://rs.tdwg.org/dwc/dwctype/MaterialSample"
     dc:relation="http://rs.tdwg.org/dwc/terms/type-vocabulary/#MaterialSample"
     dc:description="A resource describing the physical results of a sampling (or subsampling) event. In biological collections, the material sample is typically collected, and either preserved or destructively processed.">
  <preferred>
   <term dc:source="Darwin Core" dc:title="MaterialSample" xml:lang="en"/>
   <term dc:source="Darwin Core" dc:title="Amostra" xml:lang="pt"/>
   <term dc:source="Darwin Core" dc:title="Muestra" xml:lang="es"/>
   <term dc:source="Darwin Core" dc:title="試料" xml:lang="ja"/>
  </preferred>
  <alternative>
    <term dc:title="Sample" xml:lang="en"/>
  </alternative>
 </concept>```

Including the Spanish and Japanese translations, which aren't present on your screenshot.

**Comment by RicardoOrtizG** [comment]
Hi Matt, 
The TI had a confussion and send me the wrong file, I include the rigth files in this post, but beause of your message I tried to republish again and finally it works. I don't know what happen but seems to be ok now. I want to make clear that I did't do any modification in the data, metadata or IPT. It just work now.
[vocab&ext.zip](https://github.com/gbif/ipt/files/7598237/vocab.ext.zip)
